public/dir.js

```diff
@@ -42,8 +42,10 @@
     document.getElementsByClassName('dir-upload-wrapper')[0].appendChild(shareLinkWrapper);
   };
 
-  const removeListItem = (item) => () => {
+  const removeListItem = (item) => (e) => {
     item.parentElement.remove();
+    // console.log(e.currentTarget);
+    window.alert(`${item.parentElement.firstElementChild.textContent} was deleted successfully.`)
```

Comment:
This is ok for now, but a dom based confirmation is a lot more pleasant.

Reply:
Yeah, I don't know if you saw it but I asked on slack if you had any preferences regarding how we should give the user the feedback after deletion (since we have no mockup for this), and if it was OK to just alert it for now. 

I'd do a DOM based feedback, but I wasn't sure about what you and João had in mind for its design. Just doing what I think looks good might have not been enough, and an overall waste of time - at least, that is what I thought...